### ISSUE TITLE
Documentation:  Replace link with new EasyCLA links

### ISSUE THREAD

**Issue by sbtaylor15** [ortelius/ortelius#334]
On https://docs.ortelius.io/guides/contributorguide/things-to-know/things-to-know/#sign-the-cla need to reference LFX EasyCLA Tools link.

**Comment by jayesh-srivastava** [comment]
Hi. Is this the link by which the original links are to be replaced - https://easycla.lfx.linuxfoundation.org/#/
And what all links need to be replaced?

**Comment by sbtaylor15** [comment]
@jayesh-srivastava  yes, that is the correct link.

This section on the page needs to be updated to reference the new link and the new procedures to use the EasyCLA instead of emailing the PDF to us.

```
Sign the CLA
If you would like to contribute code, documentation content, blogs or graphics you will need to complete the Linux Foundation Ortelius individual Contributor Agreement(CLA). Download and sign the agreement and email it to cla@ortelius.io.

If your organization would like to become a contributing organization, please have the appropriate individual complete the Ortelius Corporate Contributor License Agreement. We welcome organizations of any size to be part of solving the microservice puzzle and encourage you to get your organization involved. Please email the completed agreement to cla@ortelius.io.

```

**Comment by jayesh-srivastava** [comment]
Hi @sbtaylor15 I have made the changes and made the PR, please do tell me if there's anything wrong with the changes made. I am not totally sure that the changes made are okay.

**Comment by sbtaylor15** [comment]
Closed ortelius/ortelius-docs#54
